Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1170,7 +1170,7 @@ def collect_message_statements(messages, message_statement_function):
             result.append('    UNUSED_PARAM(decoder);\n')
             result.append('    UNUSED_PARAM(connection);\n')
             result.append('#if ENABLE(IPC_TESTING_API)\n')
-            result.append('    if (connection.connection().ignoreInvalidMessageForTesting())\n')
+            result.append('    if (connection.protectedConnection()->ignoreInvalidMessageForTesting())\n')
```

Comment:
you could introduce StreamClientConnection::ignoreInvalidMessageForTesting instead..

Reply:
I also tried this, but `ignoreInvalidMessageForTesting()` is also an implementation detail, and also it should only be used in rare circumstances (unexpected messages) at which point I think the cost of an extra ref+deref doesn't matter.